packages/debug-ui/scripts/tcrActions.ts

```diff
@@ -0,0 +1,27 @@
+import { Civil } from "@joincivil/core";
+import BN from "bignumber.js";
```

Comment:
FYI: I believe this is the proper way to import from bignumber, as oposed to { BigNumber } which doesn't allow construction of bignumbers. No changes needed, you might find it useful thogh